Automattic/sensei
# Fix visibility of quiz passmark percentage field

Fixes #3682.

### Changes proposed in this Pull Request

Fixes the visibility of the _Quiz passmark percentage_ field.

### Testing instructions

* Open a lesson.
* In _Quiz Settings_ check the _Pass required to complete lesson_ checkbox.
* Ensure the _Quiz passmark percentage_ field is visible.
* Uncheck the _Pass required to complete lesson_ checkbox.
* Ensure the _Quiz passmark percentage_ field is hidden.
